docs/tutorials/gettingstartedexample/gse-create-tdms.md

```diff
@@ -0,0 +1,193 @@
+    
+6. Run the VI and look at the **Actual Frame Rate (fps)** indicator. It shows 40 fps.
+    
+7. Double click the TDMS File Viewer VI in the LabVIEW project.
+    
+    ![Open TDMS File Viewer VI](../../images/PXIe-148X-GSE-TDMSFileViewer.png)
+
+8. Select the correct location for the **TDMS File Directory** control. By default, TDMS files generated were saved to <font face = "courier new">\<yourprojectdir\>\\Host\\Gen\\TDMS Files</font>.
+
+    > Note: The directory browse window will not show the TDMS files located in the "TDMS Files" directory.
+    
+9. Run the TDMS File Viewer VI and verify you have one frame of data. You should have a Frame Start packet, followed by 1080 RAW12 packets, followed by a Frame End packet.
+    
+## Using line sync packets
+This tutorial will enable line sync packets in the generated data and use the TDMS file viewer to verify that each RAW12 Long Packet is proceded by a Line Start packet and followed by a Line End packet.
+    
+> Note: For the purposes of this tutorial, all input control values not specified should be left as the default value.
+    
+1. Double click the Create CSI-2 Packet TDMS Files VI in the LabVIEW project.
+
+2. Set the **Number of Frames** control to 1.
+    
+3. Set the **desired frame rate (fps)** to 1000.
+    
+4. Run the VI and look at the **Actual Frame Rate (fps)** indicator and note the FPS displayed.
+
+5. Set the **include line sync packets** control to true.
+    
+6. Run the VI and look at the **Actual Frame Rate (fps)** indicator and note the FPS displayed is slightly lower than before. This is because there are more packets being sent and more minimum delays being added to each frame.
+    
+7. Double click the TDMS File Viewer VI in the LabVIEW project.
+    
+8. Select the correct location for the **TDMS File Directory** control. By default, TDMS files generated were saved to <font face = "courier new">\<yourprojectdir\>\\Host\\Gen\\TDMS Files</font>.
+    
+9. Run the TDMS File Viewer VI and verify that all of the RAW12 Long Packets are surrounded by Line Start and Line End packets. The display should look like the image below.
+    
+    ![Viewer Line Sync Packets](../../images/PXIe-148X-TDMSFileViewer-LineSyncPackets.png)
+    
+
+## Evenly spacing long packets to fill the entire time window for a frame    
+This tutorial is going to show you how to adjust the interpacket delay to spread the line packets out evenly throughout the frame period. You will use the TDMS file Viewer to look at the packets and see how you can adjust the timing to achieve a 30 FPS TDMS file.
+    
+> Note: For the purposes of this tutorial, all input control values not specified should be left as the default value.
+    
+1. Double click the Create CSI-2 Packet TDMS Files VI in the LabVIEW project.
+
+2. Set the **Number of Frames** control to 2.
```

Comment:
Add tiny comment on why you are setting this to 2 (to clarify that frame period is not 'frame start to frame end, but rather frame start(n) to frame start(n+1)' so we need a second frame to look at.